proxy/director.go

```diff
@@ -126,7 +126,7 @@ func timedUnavailabilityFunc(wait time.Duration) func(*endpoint) {
 	return func(ep *endpoint) {
 		time.AfterFunc(wait, func() {
 			ep.Available = true
-			log.Printf("proxy: marked endpoint %s available", ep.URL.String())
+			log.Printf("proxy: marked endpoint %s available to give it another chance", ep.URL.String())
```

Comment:
@robszumski Any suggestion on the log? We set unavailable endpoint available after some failure-wait time, to give it a new chance.